examples/cfq/models.py

```diff
@@ -103,24 +107,22 @@ class MultilayerLSTM(nn.Module):
   def apply(self,
             num_layers: int,
             horizontal_dropout_masks: jnp.array,
-            vertical_dropout_masks: jnp.array,
-            input: jnp.array, previous_states: List):
+            dropout_rate: float,
```

Comment:
Why don't you name this `vertical_dropout_rate`?

Reply:
I was trying to be consistent with the names from recurren.py but probably I should either change them everywhere or not. Will modify.